packages/trail-core/test/index.test.js

```diff
@@ -105,22 +105,40 @@ describe('TrailsManager', () => {
 
       const ids = await Promise.all(records.map(r => this.subject.insert(r)))
 
-      expect((await this.subject.search({ from: '2018-01-01T11:00:00+00:00', to: '2018-01-04T13:34:56+00:00', who: 'dog', sort: 'when' }))
+      var trail = await this.subject.search({ from: '2018-01-01T11:00:00+00:00', to: '2018-01-04T13:34:56+00:00', who: 'dog', sort: 'when' })
+      expect(trail).to.be.object()
+      expect(parseInt(trail.count)).to.equal(2)
```

Comment:
why parseInt?

Reply:
the trail.count comes as string in this expect() function and in other test cases too..

Reply:
why? it should not come as a string

Reply:
I had seen similar parseInt implementation in tests in `trail/core` too ... Did not really investigate.

Reply:
understood but this is important because it's part of the interface of the API, so it makes quite a big difference whether this is a string or a number. it should be a number

Reply:
either that or, see https://github.com/brianc/node-postgres/pull/427

Reply:
This definitely is a better approach. Adding the following
```
require('pg').defaults.parseInt8 = true
```
at this place
https://github.com/nearform/trail/blob/b8387dcf7b73b276d12090f7b1cfe9ef52427c8f/packages/trail-core/lib/index.js#L4-L6

However this will change the default behaviour. Do you foresee any other queries getting effected by this ?